test/livebook_web/live/home_live_test.exs

```diff
@@ -136,6 +136,23 @@ defmodule LivebookWeb.HomeLiveTest do
       refute render(view) =~ id
     end
 
+    test "allows download the source of an existing session", %{conn: conn} do
+      {:ok, session} = Sessions.create_session()
+      Session.set_notebook_name(session.pid, "My notebook")
+
+      {:ok, view, _} = live(conn, "/")
+
+      {:error, {:redirect, %{to: to}}} =
+        view
+        |> element(~s{[data-test-session-id="#{session.id}"] a}, "Download Source")
```

Comment:
```suggestion
        |> element(~s{[data-test-session-id="#{session.id}"] a}, "Download source")
```

Reply:
Good catch! I wonder how the tests passed with this mistake?

Reply:
Nono, I changed the original text too just above, only then their failed, so it's all good!